config/config-mattermod.default.json

```diff
@@ -82,7 +82,7 @@
     "CLAGithubStatusContext": "",
     "SignedCLAURL": "",
     "NeedsToSignCLAMessage": "",
-    "BlacklistPaths": [],
+    "BlocklistPaths": [],
```

Comment:
Should be kept backwards compatible and we do the cleanup in https://github.com/mattermost/mattermost-mattermod/pull/144?

Reply:
I can rename to `BlockListPathsGlobal`